compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -72,6 +72,14 @@ namespace {
 struct CachedBlock {
   static constexpr u16 CacheIndexMax = UINT16_MAX;
   static constexpr u16 InvalidEntry = CacheIndexMax;
+  //   * MaxCachePagesToRelease default is currently 4
+  //        - We arrived at this value after noticing that mapping
+  //        in larger memory regions performs better than releasing
+  //        memory and forcing a cache hit. According to the data,
+  //        it suggests that beyond 16KB, the release execution time is
+  //        longer than the map execution time. In this way, the default
+  //        is dependent on the platform.
+  static constexpr uptr MaxCachePagesToRelease = 4U;
```

Comment:
The reason why I wanted to include `Cache` in the name was because `MaxUnusedCachePages` does and this is also used in the retrieval code. Would `MaxUnusedCachePagesToRelease` work?

Reply:
Sounds good

Reply:
I'm not sure if "Commited" is good for here (we are not sure if it's committed). Another candidate can be, `MaxReleasedCachePages` and `MaxUnreleasedCachePages`, given that the API is using "release", this seems to align with it.

Reply:
I agree, those sound good.